src/button-dropdown/utils/use-button-dropdown.ts

```diff
@@ -75,13 +75,13 @@ export function useButtonDropdown({
       target: isLink ? getItemTarget(item) : undefined,
       checked: isCheckbox ? !item.checked : undefined,
     };
+    onReturnFocus();
```

Comment:
yes: previously, if a consumer tried to move the focus in the event handler, that would then get overridden because this logic ran `onReturnFocus` _after_ the handler. This way that can be controlled more effectively